### ISSUE TITLE
specialized static GADMarketplaceKitSignals.appDistributor() on iOS for Mac

### ISSUE THREAD

**Issue by khoogheem** [googleads/swift-package-manager-google-mobile-ads#94]
In Version 11.7.0

```
3  libswiftCore.dylib             0x3a99f4 swift::fatalErrorv(unsigned int, char const*, char*) + 126
4  libswiftCore.dylib             0x3a9a14 swift::fatalError(unsigned int, char const*, ...) + 30
5  libswiftCore.dylib             0x3db5e4 swift::ResolveAsSymbolicReference::operator()(swift::Demangle::__runtime::SymbolicReferenceKind, swift::Demangle::__runtime::Directness, int, void const*) + 604
6  libswiftCore.dylib             0x4103f4 swift::Demangle::__runtime::Demangler::demangleSymbolicReference(unsigned char) + 228
7  libswiftCore.dylib             0x40c7b8 swift::Demangle::__runtime::Demangler::demangleType(__swift::__runtime::llvm::StringRef, std::__1::function<swift::Demangle::__runtime::Node* (swift::Demangle::__runtime::SymbolicReferenceKind, swift::Demangle::__runtime::Directness, int, void const*)>) + 272
8  libswiftCore.dylib             0x3e3914 swift_getTypeByMangledNameImpl(swift::MetadataRequest, __swift::__runtime::llvm::StringRef, void const* const*, std::__1::function<void const* (unsigned int, unsigned int)>, std::__1::function<swift::TargetWitnessTable<swift::InProcess> const* (swift::TargetMetadata<swift::InProcess> const*, unsigned int)>) + 504
9  libswiftCore.dylib             0x3dda84 swift_getTypeByMangledName + 836
10 libswiftCore.dylib             0x3ddecc swift_getTypeByMangledNameInContextImpl(char const*, unsigned long, swift::TargetContextDescriptor<swift::InProcess> const*, void const* const*) + 172
11 XXX                      0xc300 __swift_instantiateConcreteTypeFromMangledName (<compiler-generated>)
12 XXX                      0xfdfca8 specialized static GADMarketplaceKitSignals.appDistributor()
```

**Comment by olekenneth** [comment]
Same as #97

**Comment by malandr2** [comment]
Hi @all,

This issue has been resolved in [11.9.0](https://developers.google.com/admob/ios/rel-notes). Please update your Google Mobile Ads SDK to the latest version to see this change.
